src/main/java/org/springframework/data/elasticsearch/core/DefaultReactiveIndexOperations.java

```diff
@@ -304,6 +309,27 @@ public IndexCoordinates getIndexCoordinates() {
 		return (boundClass != null) ? getIndexCoordinatesFor(boundClass) : boundIndex;
 	}
 
+	@Override
+	public Flux<IndexInformation> getInformation() {
+		org.elasticsearch.client.indices.GetIndexRequest getIndexRequest = requestFactory.getIndexRequest(getIndexCoordinates());
+
+		return Mono.from(operations.executeWithIndicesClient(client -> client.getIndex(HttpHeaders.EMPTY, getIndexRequest)
+				.flatMap(getIndexResponse -> getAliasesForIndex(getIndexCoordinates().getIndexNames()).map(aliases -> {
```

Comment:
no separate call to get the alias information. This is returned in the GetIndex call